src/composables/trade/useTradeState.ts

```diff
@@ -0,0 +1,35 @@
+import { readonly, ref } from 'vue';
+
+// globals
+const tokenInAddress = ref('');
+const tokenInAmount = ref('');
+const tokenOutAddress = ref('');
+const tokenOutAmount = ref('');
```

Comment:
Great that you extracted this into a composable, but think we can make it cleaner using a reactive object.
```suggestion
const state = reactive({
    tokenInAddress: '',
    tokenInAmount: '',
    tokenOutAddress: '',
    tokenOutAmount: ''
});
```

Reply:
yeah